src/couch_prometheus/src/couch_prometheus_server.erl

```diff
@@ -221,36 +223,155 @@ get_message_queue_stats() ->
         to_prom(erlang_message_queue_size, gauge, "size of message queue", QueueLenByLabel)
     ].
 
-message_queue_len(undefined) ->
-    0;
-message_queue_len(Pid) when is_pid(Pid) ->
-    case erlang:process_info(Pid, message_queue_len) of
-        {message_queue_len, N} ->
-            N;
-        _ ->
-            0
-    end.
```

Comment:
`message_queue_len` is no longer required because we get the message queue stats from `chttpd_node:message_queues/0` instead.